onnx/defs/math/defs.cc

```diff
@@ -496,3 +496,36 @@ OPERATOR_SCHEMA(MatMul)
     .SetDoc(R"DOC(
 Matrix product that behaves like numpy.matmul: https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.matmul.html
 )DOC");
+
+OPERATOR_SCHEMA(TopK)
+    .SetDoc(R"DOC(
+Retrieve the top-K elements for the last dimension. Given an input tensor of
+shape [a_1, a_2, ..., a_n, r] and integer argument k, return two outputs:
+  -Value tensor of shape [a_1, a_2, ..., a_n, k] which contains the values of
+   the top k elements along the last dimension
+  -Index tensor of shape [a_1, a_2, ..., a_n, k] which contains the indices
+   of the top k elements (original indices from the input tensor).
+
+Given two equivalent values, this operator uses the indices along the last dim-
+ension as a tiebreaker. That is, the element with the lower index will appear
+first.
+)DOC")
+    .Input(0, "X", "Tensor of shape [a_1, a_2, ..., a_n, r]", "T")
+    .AllowConsumed({{2, 0}})
```

Comment:
I think this means the Input[2] is allowed to be consumed by Output[0], and that's shouldn't be what you want :-)